app/controllers/host_controller.rb

```diff
@@ -645,20 +646,18 @@ def get_hosts
     @host_pages, @hosts = paginate(:hosts, :per_page => @items_per_page, :order => @col_names[get_sort_col] + " " + @sortdir)
   end
 
+  def title
+    _("Host")
+  end
+
   def get_session_data
-    @title      = _("Hosts")
-    @layout     = "host"
+    super
     @drift_db   = "Host"
-    @lastaction = session[:host_lastaction]
-    @display    = session[:host_display]
-    @filters    = session[:host_filters]
-    @catinfo    = session[:host_catinfo]
   end
 
   def set_session_data
+    super
     session[:host_lastaction] = @lastaction
```

Comment:
Sorry, one more thing, can you also remove this (`lastaction`) line? That one is handled by the mixin too.

Also, the same for `catinfo`.